addons/point_of_sale/models/pos_session.py

```diff
@@ -186,8 +186,6 @@ def create(self, values):
         if values.get('name'):
             pos_name += ' ' + values['name']
 
-        uid = SUPERUSER_ID if self.env.user.has_group('point_of_sale.group_pos_user') else self.env.user.id
```

Comment:
it changes the existing logic that only does the sudo if you  have point of sale user group

Reply:
I don't even get what is the logic of this ??
Pos Users can create bank statements and Pos Sessions, so there shouldn't be a need for the sudo anyway (or at least not here).  If we only want to give sudo abilities to user able to create a pos session, we could just check the access rights at the beginning of the method:
But I could add a check on the creation rights on the beginning of the function if you want to be sure we don't give too much rights to the user
```python3
self.check_access_rights('create')
self.env['account.bank.statement'].check_access_rights('create')
```
 at the beginning of the create if you want to see if user has the rights to create for the two models.

Reply:
The question here is not if it is logical or not. The behavior was changed by your PR. You can see here that it has been introduced by https://github.com/odoo/odoo/commit/a29e291b69335260b26840c195599ed5bf845e16 because pos users don't have access to bank statements.

As it is not a bug, I won't do the modification in stable version. But you can do a PR in master or  a feedbacki

Reply:
Ok, I missed the ir.rule restricting the creation ^^.

Reply:
Fixed to keep the old logic.